src/Containers/Microsoft.NET.Build.Containers/ContainerBuilder.cs

```diff
@@ -91,7 +102,7 @@ public static class ContainerBuilder
         {
             if (isDaemonPush)
             {
-                LocalDocker localDaemon = GetLocalDaemon(localContainerDaemon,Console.WriteLine);
+                LocalDocker localDaemon = GetLocalDaemon(localContainerDaemon, WriteLogMessage);
```

Comment:
minor: passing and `ILogger` or `ILoggerFactory` will allow recipients to decide the level of the logged message.